README.md

```diff
@@ -274,9 +279,22 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 "state_timeout_in_seconds": 600,
 
                 // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account
+                // OIDC user to an iRODS user account.
+                // Only 'irods_user_claim' or 'user_attribute_mapping' can be
+                // declared at once.
```

Comment:
Please update the documentation to follow the same formatting/style as `/irods_client/tls/client_server_policy`.

List the supported options and give a brief description explaining what they do and/or how they change the interface.

Reply:
Does this make sense in this case? claim & mapping are not "values", they're the actual "keys".

Is there a better format for this case?

Reply:
It's more about making the docs consistent. All properties accepting a range of values have docs which list the acceptable values. Maintaining that is good IMO.

A better example of this is the log_level property.

> claim & mapping are not "values", they're the actual "keys".

It's not clear to me why that's important. Please elaborate.

Reply:
I think it would be confusing, since the format is used throughout for 'values'. Throwing it in for 'keys' would get you if you don't look closely at it.

Maybe we're talking a bit past each other here? Not sure if my 'keys' and 'values' are clear here.

Reply:
The docs for the property need to include all of the information for a user/admin to understand how to use it. One could argue that everything listed in the json file is a value. It depends on how you look at it. As long as the docs are clear, there's no issue.

```js
// Defines how user mapping is carried out.
//
// The following values are supported:
// - irods_username         
// - user_attribute_mapping
//
// When set to "irods_username", the HTTP API will ...
//
// When set to "user_attribute_mapping", the HTTP API will ...
"irods_user_claim": "irods_username",
```

That is just an example. The docs make it clear what is allowed and then proceeds to explain what the HTTP API does when set to either of the values.

Whether the value represents a key is irrelevant because that may not be true in the future.

Does that make sense or am I misunderstanding?

Reply:
Still think we're not there yet.

We're talking about two separate 'keys', `irods_user_claim` and `user_attribute_mapping`:
```json
"irods_user_claim": "COOL_USERNAME_HERE",

"user_attribute_mapping": {
  "COOL_USERNAME_HERE": {
    "not": "important_right_now",
    "minor": "details_here"
}
```

Only one of them can be active at a time.

---

Mini discuss with Kory:

For the time being, we're just going to document the options individually, and briefly mention that the options cannot be used together.

Reply:
Done. Did a minor reword, let know if there's a better sentence.